x-pack/plugin/src/test/resources/rest-api-spec/test/ml/inference_crud.yml

```diff
@@ -871,3 +871,14 @@ setup:
   - is_false: trained_model_configs.0.estimated_heap_memory_usage
   - is_false: trained_model_configs.0.estimated_operations
   - is_false: trained_model_configs.0.license_level
+---
+"Test deprecation of include model definition param":
+  - skip:
+      features: "warnings"
+  - do:
+      warnings:
+        - "[include_model_definition] parameter is deprecated! Use parameter [include] and supply the flag 'definition' instead."
```

Comment:
I'm wondering if the wording here could be more direct and say sth like: "Use [include=definition] instead".

Reply:
I can update the message, for sure.